### PR TITLE
Indonesia payment method

### PR DESCRIPTION
**Pull request by candrasaputra**
** Checklist **
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [ ] I have provided a link to the brand icon’s brand guidelines whenever possible.

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

**Review by tauthomas01 — CHANGES_REQUESTED** [review]
Hi @candrasaputra,

We are no longer accepting icons that are using `base64` because it's [too heavy](https://github.com/activemerchant/payment_icons/issues/407).

Do you have by any chance the SVG files with proper semantics? (`<path>`, `<rect>`, `<circle>`...)

**Comment by candrasaputra** [comment]
@tauthomas01 not yet, but we work on it.

**Comment by candrasaputra** [comment]
@tauthomas01 i has update the payment icon, please help to review. thank you.

File: app/assets/images/payment_icons/bri_direct_debit.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...85.435c-.415 0-.757-.127-1.025-.382-.268-.254-.408-.596-.408-1.024 0-.429.14-.77.415-1.025.275-.255.603-.382.978-.382s.696.114.957.342c.262.228.395.536.395.93v.41h-1.989c.027.147.1.267.235.368a.76.76 0 00.449.14.878.878 0 00.663-.268l.415.456zm-.73-1.621a.589.589 0 00-.382-.134.73.73 0 00-.415.134.524.524 0 00-.221.368h1.199a.514.514 0 00-.181-.368zM30.235 13.585c.334 0 .622.128.87.389s.369.59.369.998c0 .408-.128.743-.382 1.018-.255.274-.543.408-.864.408-.322 0-.59-.134-.804-.401v.361h-.763v-3.757h.763v1.34c.208-.235.476-.356.81-.356zM29.41 15c0 .22.06.401.187.542a.57.57 0 00.449.208.618.618 0 00.462-.208c.134-.14.201-.321.201-.542a.838.838 0 00-.194-.556.59.59 0 00-.462-.221.577.577 0 00-.456.22.833.833 0 00-.187.557zM32.203 13.284a.43.43 0 01-.127-.315.42.42 0 01.127-.314.43.43 0 01.315-.128c.128 0 .228.04.315.128a.43.43 0 01.127.314.42.42 0 01-.127.315.43.43 0 01-.315.127.405.405 0 01-.315-.127zm.697 3.08h-.763v-2.732h.763v2.733zM34.836 14.181v1.226c0 .107.026.188.08.248a.25.25 0 00.194.087c.14 0 .261-.067.362-.194l.281.536c-.234.214-.489.321-.75.321a.92.92 0 01-.663-.261c-.18-.174-.275-.409-.275-.703v-1.26h-.321v-.549h.321v-.817h.764v.817h.663v.55h-.656z" fill="#F37021"/></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/bri_direct_debit.svg:1]: Can we remove the extra white background (`<path xmlns="http://www.w3.org/2000/svg" fill="#fff" d="M0 0h38v24H0z"/>`)? We're not seeing the grey border

[Video demonstration](https://screenshot.click/24-18-o5kep-pvgd8.mp4)

**Review by tauthomas01 — CHANGES_REQUESTED** [review]
Hi @candrasaputra ,

Thank you for the fixes, the SVG looks very good now 👍 

On every icons, can we remove the extra white background (`<path xmlns="http://www.w3.org/2000/svg" fill="#fff" d="M0 0h38v24H0z"/>`)? We're not seeing the grey border.

**Comment by candrasaputra** [comment]
@tauthomas01 done

**Review by tauthomas01 — APPROVED** [review]
Looks good.
Thank you for the changes

**Comment by candrasaputra** [comment]
@tauthomas01 thanks.

could i know, when i can using this icon in shopify?

**Comment by larouxn** [comment]
👋 @candrasaputra, I'd estimate the icon will be live by end of next week.

**Comment by candrasaputra** [comment]
@larouxn cool, thanks.

**Pull request merged**